lua/entities/gmod_tardis/modules/sh_player.lua

```diff
@@ -0,0 +1,15 @@
+if SERVER then
+	ENT:AddHook("StopMat", "EnterOnTP", function(self)
```

Comment:
hook names should be prefixed with the module 

```suggestion
	ENT:AddHook("StopMat", "player-enterontp", function(self)
```